### PR TITLE
MWPW-135816 enable ability to get forced VAT exclusive prices 

### PR DESCRIPTION
**Pull request by npeltier**
1. passes along the `forceTaxExclusive` in OST links & generated spans
2. added unit test (over factorized price span test in separate commit)

**Test URLs:**
check `Temporary test of VAT exclusive` section, 2nd price. Note if you want to play around with it, that you need the [branch version of OST](https://mwpw-135816--milo--npeltier.hlx.page/tools/ost?...&token=<token>). Finally note that re-edit is globally broken tracked https://jira.corp.adobe.com/browse/MWPW-137106 for this
 
- Before: https://main--milo--adobecom.hlx.page/fr/drafts/npeltier/plans-card
- After: https://mwpw-135816--milo--npeltier.hlx.page/fr/drafts/npeltier/plans-card

### CODE REVIEW

File: test/blocks/merch/merch.test.js [hunk 1]
```diff
@@ -17,6 +17,32 @@ const config = {
...
+    expect(dataset.template).to.be.undefined;
+  }
+  Object.keys(expectedAttributes).forEach((key) => {
+    const value = expectedAttributes[key];
+    if (value === 'UNDEF') {
+      expect(dataset[key]).to.be.undefined;
+    } else {
+      expect(dataset[key]).to.equal(value);
+    }
+  });
```
Comment (vladen) [inline test/blocks/merch/merch.test.js:43]: The `.equal` Chai assertion should be able to work with the regular `undefined` value as well, per https://www.chaijs.com/api/bdd/#method_equal.

Based on this, the `'UNDEF'` strings do not seem to be necessary:
```
Object.entries(expectedAttributes).forEach(([key, value]) => {
      expect(dataset[key]).to.equal(value);
});
// validatePriceSpan('.merch.price.term', { displayRecurrence: undefined });
```

File: libs/blocks/merch/merch.js [hunk 2]
```diff
@@ -75,6 +75,7 @@ export async function getPriceContext(el, params) {
   const displayPerUnit = params.get('seat');
   const displayRecurrence = params.get('term');
   const displayTax = params.get('tax');
+  const forceTaxExclusive = params.get('forceTaxExclusive');
```
Comment (vladen) [inline libs/blocks/merch/merch.js:78]: Can we continue using shortened names for params in OST URLs, like `old`, `seat`, `term`, and `tax`?
Maybe `excl`?

Comment (yesil) [reply libs/blocks/merch/merch.js:78]: `exclusive` is fine as well

Comment (3ch023) [reply libs/blocks/merch/merch.js:78]: or 'notax'?

Comment (npeltier) [reply libs/blocks/merch/merch.js:78]: should i create a poll 😁

**Comment by npeltier** [comment]
here you are @vladen

**Comment by 3ch023** [comment]
@npeltier label check was failing, I added 'needs-verification'. @afmicka or one of us should test it before we merge.

**Review by 3ch023 — APPROVED** [review]

**Review by vladen — APPROVED** [review]

**Pull request merged**